Launcher/config_es.xml

```diff
@@ -1093,6 +1093,8 @@
     <S id="PRG_Launch_exe">sh2pc.exe</S>
     <S id="PRG_Ini_name">d3d8.ini</S>
     <S id="PRG_Ini_error">No se pudo guardar el archivo d3d8.ini.</S>
+    <S id="PRG_Lang_confirm">¿Deseas cambiar de idioma al %s?&#x0a;&#x0a;Do you want to switch to language %s?</S>
```

Comment:
This line won't work.  The string can only have a single `%s` in it otherwise it will crash.  Maybe we should remove the English text from here?

Reply:
Whoopsie-daisy. Fixed now.